app.js

```diff
@@ -60,37 +37,72 @@ let fetch = (user, repo) => {
       sort: "updated",
       order: "desc",
       since: ghCommit.commit.author.date
-    });
-  }).then((issues) => {
-    const json = {};
+    })
+  }
+}
+
+const testLabel = (l) => { return regex.test(l.name) }
+
+const fetchCommits = (user, repo) => {
+  return github.repos.getCommits({
+    user,
+    repo,
+    sha: "production"
+  })
+}
+
+const collectIssues = (issues) => {
+  return new Promise((resolve, reject) => {
```

Comment:
I just never get any issues back. No error. No matter what you search for, no issues are returned.